src/Asset/AppAsset.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Asset;
+
+use Yiisoft\Assets\AssetBundle;
+
+class AppAsset extends AssetBundle
+{
+    public ?string $basePath = '@basePath';
+    public ?string $baseUrl = '@assetsUrl';
+    public ?string $sourcePath = __DIR__ . '/css';
```

Comment:
I think asset files shouldn't be in the `src` directory. Likely in `root/assets` or `@resources/assets`.

Reply:
done.